client/src/components/SortDirection/SortDirection.stories.js

```diff
@@ -0,0 +1,19 @@
+import _ from "lodash";
+import React from "react";
+
+import { SortDirection } from "./SortDirection";
+
+export default {
+  title: "SortDirection",
+  component: SortDirection,
+};
+
+const Template = (args) => <SortDirection {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  className: "",
+  style: {},
+  active: true,
+  sortDirection: "",
```

Comment:
Actually, `SortDirection` doesn't have `className`, `style` props!

If you look at `SortDirection.js`, you can see the props it takes:
```
export const SortDirection = ({ sortDirection, active }) => ...
```
The above is "destructuring" the props. (you can read about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)) So this is equivalent as writing:
```
export const SortDirection = (props) => {
  const { sortDirections, active } = props;
  ...
```
And as you can see, `SortDirection` isn't using any `className` or `style` from props